troubadix/runner.py

```diff
@@ -126,7 +126,7 @@ def _process_plugin_results(
             plugin_results,
         ) in results.items():
             if plugin_results or self.debug:
-                self._report_info(f"Running plugin {plugin_name}")
+                self._report_info(f"Results for plugin {plugin_name}")
```

Comment:
We could also make this even better. How about writing `f"No results for plugin {plugin_name}"`

```python
if plugin_results:
  self._report_info(f"Results for plugin {plugin_name}")
elif self.debug:
  f"No results for plugin {plugin_name}"
``` 
  
What do you think?

Reply:
Would be nice 👍🏻